### ISSUE TITLE
Linux cloud server not working? yet?

### ISSUE THREAD

**Issue by yppeno** [hacksider/Deep-Live-Cam#765]
Hi guys
I installed on a cloud linux server but there is not interface loading when i launch python run.py --execution-provider cuda

terminal says:" 
```
   self.tk = _tkinter.create(screenName, baseName, className, interactive, wantobjects, useTk, sync, use)
_tkinter.TclError: no display name and no $DISPLAY environment variable
```

Can you do something to have a display when run on a cloud linux on be able to use webcam on cloud too?

Thanks

**Comment by yppeno** [comment]
Anyone? :)

**Comment by RudideC** [comment]
Yeah, you kinda need a webcam and a screen to use it. Run it on your own computer

**Comment by RudideC** [comment]
You could theoretically offload all the work to the linux server, but you'll need to change the code for that

**Comment by fanqiangwei** [comment]
you  can  try this:
     python run.py --source=./111.jpeg --target=./222.jpeg --output=./output

**Comment by luna-nightbyte** [comment]
This would be really interesting tho. A server side version, and a client side version to run the UI, send sources + frames from webcam and eventually receive the results (image/video files and live frames).

I would ofc not except good fps on live streams, but it could be nice as a external processor. I have a few better gpu's myself on a server without display.

**Comment by hacksider** [comment]
This is not planned as of this moment. Closing this. But I believe there's a fork which have this feature
